reference/generators/visualstudio.rst

```diff
@@ -24,30 +24,45 @@ Generated xml structure:
         <Conan-Lib2-Root>{PACKAGE LIB2 FOLDER}</Conan-Poco-Root>
         ...
       </PropertyGroup>
-      <PropertyGroup Label="ConanVariables">
```

Comment:
`/home/travis/build/conan-io/docs/reference/generators/visualstudio.rst:16:Could not lex literal_block as "xml". Highlighting skipped.`